hieradata/common.yaml

```diff
@@ -524,7 +524,6 @@ govuk_ci::master::pipeline_jobs:
   rack-logstasher: {}
   rails_translation_manager: {}
   router-data: {}
-  rubocop-govuk: {}
   scss-lint-govuk: {}
   seal: {}
```

Comment:
```suggestion

```

The Seal was switched to GitHub Actions in https://github.com/alphagov/seal/pull/162.